src/core/assets/copy/IOdsCoreCopy.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Interface representing the structure of copy texts used in various parts of the ODS Core package.
+ * Each property in the interface corresponds to a specific component or feature, containing the necessary
+ * text labels or functions that return text strings.
+ */
+
+export interface IOdsCoreCopy {
```

Comment:
What do you think if instead we remove this interface, rename `enCopy` to `coreCopy` and use `typeof coreCopy` on the core provider? In that way we won't need to maintain the interface when adding changes to the coreCopy

Reply:
Good idea 👍, updated.